spaces/domains.json

```
@@ -1,4 +1,6 @@
 {
+"gov.dogownertoken.com": "dogown.eth"
+}
```

Please remove the extra bracket and add a comma.
```suggestion
"gov.dogownertoken.com": "dogown.eth",
```